RadixWallet/Clients/UserDefaults+Dependency+Extension/UserDefaults+Dependency+Extension.swift

```diff
@@ -10,6 +10,9 @@ public enum UserDefaultsKey: String, Sendable, Hashable, CaseIterable {
 	case transactionsCompletedCounter
 	case dateOfLastSubmittedNPSSurvey
 	case npsSurveyUserID
+	case didMigrateKeychainProfiles
+	case lastCloudBackups
+	case lastManualBackups
```

Comment:
Do we need the `last` prefix for these? At first I thought we were only setting the last one, but as they correspond to a dictionary of `[UUID: BackupResult]` I would just name them:
```Swift
case cloudBackups
case manualBackups
```

Reply:
It only gets the last one, the ID corresponds to the profile